### DISCUSSION TITLE
React-Joyride spotlight

### DISCUSSION THREAD

**Discussion by Vaishnavi9** [gilbarbara/react-joyride#625] [Q&A] [answered]
Hello Gilbarbara,

When I override the spotlight back-ground color to white the contents also become white. Any solution? I wanted the spotlight to be pure white.

**Comment by gilbarbara** [comment, marked as answer]
hey @Vaishnavi9 

That's probably related to mix-blend-mode.
Please check this [issue](https://github.com/gilbarbara/react-joyride/issues/411)
